nixos/modules/services/networking/radicale.nix

```diff
@@ -41,7 +41,7 @@ in
     };
 
     services.radicale.config = mkOption {
-      type = types.string;
+      type = types.str;
```

Comment:
```suggestion
      type = types.lines;
```

Reply:
I thought about that, but concatenation doesn't have that great behavior for ini-like files (which radicale uses)

Reply:
yeah, just checked - it doesn't perform merge